src/server.js

```diff
@@ -13,6 +13,35 @@ import {collectDependencyData} from './collectDependencyData.js';
 import {collectMetadata} from './collectMetadata.js';
 import * as fsStore from './fs-store.js';
 
+export type IntrospectionSchema = {
+  version: string,
+  server: Dependencies,
+  browser: Dependencies,
+  runtime: Metadata,
+};
+export type Dependencies = {
+  timestamp: number,
+  dependencies: Array<Dependency>,
+  enhanced: Array<{name: string}>,
+};
+export type Dependency = {
+  name: string,
+  stack: string,
+  dependencies: Array<string>,
+};
+export type Metadata = {
+  timestamp: number,
+  pid: number,
+  nodeVersion: string,
+  npmVersion: string,
+  yarnVersion: string,
+  lockFileType: string,
+  dependencies: {[string]: string},
+  devDependencies: {[string]: string},
+  varNames: [string],
```

Comment:
Just wanted to double check that the type here is a single-element array of strings.  If it should allow multiple elements, this could be be `Array<string>` or `string[]` instead.

Reply:
Oh, good catch, that should have been a Array<string>